packages/react-components/src/components/Button/NavButton.jsx

```diff
@@ -0,0 +1,48 @@
+import { forwardRef } from 'react'
+import { func, bool, string, oneOf } from 'prop-types'
+import BaseButton from './BaseButton'
+import theme from '../theme'
+
+const NavButton = forwardRef(
+  ({ disabled, onClick, variant, title, type, isActive, ...props }, ref) => {
```

Comment:
I don't think we need `variant` or `type` here - those seem more applicable to the normal button we use throughout the app (`variant` is like `primary`, `secondary`...etc)

Reply:
Hmm maybe. My understanding of the primary and secondary etc is that they define style rules. And this button is basically the secondary style (white with the outline). Do you not think of this as a secondary button style?

Reply:
secondary style is close, but its still more rectangular than circular.